Arithmetic processor - rounding (#4621)
DataDog/documentation: content/en/logs/processing/processors.md

* The operator `-` needs to be space split in the formula as it can also be contained in attribute names.
* If the target attribute already exists, it is overwritten by the result of the formula.

> * Results are rounded up to the 9th decimal. For example, if the result of the formula is `1.1234567891`, the actual value of the attribute will be `1.123456789`.

Reviewer: ```suggestion
* Results are rounded up to the 9th decimal. For example, if the result of the formula is `0.1234567891`, the actual value stored for the attribute is `0.123456789`.
```
